include/common/scaled-scene-buffer.h

```diff
@@ -32,7 +36,8 @@ struct scaled_scene_buffer {
 	struct wl_listener destroy;
 	struct wl_listener output_enter;
 	struct wl_listener output_leave;
-	const struct scaled_scene_buffer_impl *impl;
+	struct scaled_scene_buffer_impl *impl;
+	struct wl_list link;
```

Comment:
We should say what linked list this is for. Looks like `other cached scaled-scene-buffers`.

Reply:
Added a comment:
```
	/*
	 * This holds all the created scaled-*-buffers and is referenced for
	 * sharing backing buffers
	 */
```